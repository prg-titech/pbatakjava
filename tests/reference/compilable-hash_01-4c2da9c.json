{
    "basename": "compilable-hash_01-4c2da9c",
    "cmd": "{infile}",
    "infile": "tests/hash_01/build.sh",
    "infile_hash": "a1f70bf2ef2682f6c9838a4e9fc98bb68207a94fe0419d4f634ff4fa",
    "stdout": "compilable-hash_01-4c2da9c.stdout",
    "stdout_hash": "502237a3142b2c73d71f4d11fac7c0f093a7a5aea0b9639ba9162bde",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}