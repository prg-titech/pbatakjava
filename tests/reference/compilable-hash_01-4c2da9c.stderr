tests/hash_01/2/HashValue.java:0: error: return value must be an instance of boolean which null is not
tests/hash_01/1/HashValue.java:0: error: return value must be an instance of boolean which null is not
tests/hash_01/Test.java:0: error: can not assign variable digest of type HashValue_v1 a value of type HashValue
